wiki/Ranking_Criteria/osu!mania/es.md

```diff
@@ -0,0 +1,205 @@
+# Criterios de clasificación de osu!mania
+
+***Aviso: Este artículo es una extensión de los [criterios generales de clasificación](/wiki/Ranking_Criteria).***
+
+Este conjunto de **criterios de clasificación de osu!mania** establece [reglas y pautas](/wiki/Ranking_Criteria#términos_generales) que los [beatmaps](/wiki/Beatmap) específicos de [osu!mania](/wiki/Game_mode/osu!mania) deben seguir para avanzar en el [procedimiento de clasificación de beatmaps](/wiki/Beatmap_ranking_procedure).
+
+## Glosario
+
+### Nombres de las dificultades
+
+*Artículo principal: [Nombramiento de dificultades](/wiki/Ranking_Criteria/Difficulty_naming)*
+
+- ![](/wiki/shared/diff/easy-m.png?20211215) Easy
+- ![](/wiki/shared/diff/normal-m.png?20211215) Normal
+- ![](/wiki/shared/diff/hard-m.png?20211215) Hard
+- ![](/wiki/shared/diff/insane-m.png?20211215) Insane
+- ![](/wiki/shared/diff/expert-m.png?20211215) Expert
+
+### Términos comunes
+
+- **BPM**: Tempo de una canción medido en pulsaciones por minuto.
+- **Column**: Líneas verticales en el campo de juego donde las notas son colocadas. El editor usa `0|1|2|3|4|5|6|7|8|9` para las etiquetas de marcas de tiempo.
+- **Key mode**: La cantidad de columnas presentes en una dificultad.
+- **Keysounding**: Aplicar hitsounds que son extremadamente similar o directamente de la canción en sus respectivos lugares. Este método de hitsounding generalmente proporciona poca retroalimentación al jugador.
+- **Slider velocity**: La velocidad a la que se mueven las notas/barras en el campo de juego. La velocidad base del slider (1.0x) se basa en la velocidad de desplazamiento del usuario y se pueden realizar cambios adicionales a través de puntos de tiempo heredados (verdes).
+- **Long-term slider velocity**:Un cambio de velocidad del slider que dura más de 4 compases.
+- **Musical phrase**: Una unidad independiente musical que suele ser de 4 compases de largo.
+- **Short-term slider velocity**: Un cambio en la velocidad del slider que dura menos que 1/2 de un pulso.

```

Comment:
```suggestion
- **Short-term slider velocity**: Un cambio en la velocidad del slider que dura menos que 1/2 de un tiempo.
```

Reply:
Will not change this, I use "pulso" instead of "tiempo" since that's how you say beat, if I use "tiempo" this will have another meaning completely different to what its trying to convey.